vendor/engines/split_accounts/spec/models/split_account_spec.rb

```diff
@@ -30,42 +33,61 @@
     context "when splits do not total 100 and one split has extra_penny" do
       let(:split_account) do
         build(:split_account, without_splits: true).tap do |split_account|
-          split_account.splits << build(:split, percent: 50, extra_penny: true, parent_split_account: split_account)
-          split_account.splits << build(:split, percent: 49.99, extra_penny: false, parent_split_account: split_account)
+          split_account.splits << build(:split, percent: 50, extra_penny: true, parent_split_account: split_account, subaccount: subaccount_1)
+          split_account.splits << build(:split, percent: 49.99, extra_penny: false, parent_split_account: split_account, subaccount: subaccount_2)
         end
       end
 
       it "is invalid" do
         expect(split_account).not_to be_valid
-        expect(split_account.errors).to include(:splits)
+        message = I18n.t("activerecord.errors.models.split_accounts/split_account.attributes.splits.percent_total")
+        expect(split_account.errors[:splits]).to include(message)
       end
     end
 
     context "when splits total 100 and no splits have extra_penny" do
       let(:split_account) do
         build(:split_account, without_splits: true).tap do |split_account|
-          split_account.splits << build(:split, percent: 50, extra_penny: false, parent_split_account: split_account)
-          split_account.splits << build(:split, percent: 50, extra_penny: false, parent_split_account: split_account)
+          split_account.splits << build(:split, percent: 50, extra_penny: false, parent_split_account: split_account, subaccount: subaccount_1)
+          split_account.splits << build(:split, percent: 50, extra_penny: false, parent_split_account: split_account, subaccount: subaccount_2)
         end
       end
 
       it "is invalid" do
         expect(split_account).not_to be_valid
-        expect(split_account.errors).to include(:splits)
+        message = I18n.t("activerecord.errors.models.split_accounts/split_account.attributes.splits.only_one_extra_penny")
```

Comment:
Seems like there should be a cleaner way to do this test, but I haven't seen a better way.